bokehjs/gulp/tasks/test.coffee

```diff
@@ -4,6 +4,9 @@ mocha = require "gulp-mocha"
 paths = require "../paths"
 utils = require "../utils"
 
+global.atob = require "atob"
+global.btoa = require "btoa"
```

Comment:
Those should be added in `bokehjs/test/index.coffee` next to other `global.X = ...` lines. Just merge `master` first in to this branch.